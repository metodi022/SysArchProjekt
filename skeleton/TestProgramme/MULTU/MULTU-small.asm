lui $1, 0x0000  // 32'd40965
ori $1, 0xa005
lui $2, 0x0000  // 32'd40970
ori $2, 0xa00a
multu $1, $2    // multiply $1 and $2
mfhi $3         // 0x00000000
mflo $4         // 0x64096032

// 1678336050 = 0x64096032
