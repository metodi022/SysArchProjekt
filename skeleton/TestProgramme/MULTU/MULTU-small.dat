0x3c010000
0x3421a005
0x3c020000
0x3442a00a
0x00220019
0x00001810
0x00002012
