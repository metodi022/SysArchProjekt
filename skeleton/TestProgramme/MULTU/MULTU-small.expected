@1 0000a005
@2 0000a00a
@3 00000000
@4 64096032
